src/features/Layout/Footer/components/SocialsAndLegal.tsx

```diff
@@ -7,11 +7,33 @@ import { DisclaimerModal, NewTabLink } from '@components';
 import './SocialsAndLegal.scss';
 import { useAnalytics } from '@utils';
 
-const SocialMediaLink = ({ link, text }: { link: string; text: string }) => {
+const SocialMediaLink = ({
+  link,
+  text,
+  icon
+}: {
+  link: string;
+  text: string;
+  icon: null | string;
+}) => {
   const trackSocial = useAnalytics({
     category: ANALYTICS_CATEGORIES.FOOTER
   });
 
+  if (icon) {
+    return (
+      <NewTabLink
+        className="SocialMediaLink"
+        key={link}
+        href={link}
+        aria-label={text}
+        onClick={() => trackSocialIconClicked(text, trackSocial)}
+      >
+        <img src={icon} width="18.41px" height="18px" style={{ verticalAlign: 'top' }} />
```

Comment:
Can this be `width="18px" height="18px"` ?

Reply:
Yeah - I was going to do that, but the others are 18.406/18.41px so thought I'd do the same sizing

Reply:
I just pushed up changes